chubby/chulib.py

```diff
@@ -84,5 +94,6 @@ def get_login(username: str):
         github3 login object
     """
     if not username:
+        logger.warning("Config for {} doesn't exist, creating new section...")
```

Comment:
change the `warning` to `info` and format the string.. and maybe change to `creating new config section...`